resources/views/meetings/edit.blade.php

```diff
@@ -0,0 +1,161 @@
+<x-app-medium-layout>
+    <x-slot name="title">{{ $meeting->id ? __('Edit meeting') : __('Add new meeting') }}</x-slot>
+    <x-slot name="header">
+        <ol class="breadcrumbs" role="list">
+            <li><a href="{{ localized_route('projects.my-projects') }}">{{ __('My projects') }}</a></li>
+            <li><a href="{{ localized_route('projects.manage', $project) }}">{{ $project->name }}</a></li>
+            <li><a href="{{ localized_route('engagements.manage', $engagement) }}">{{ $engagement->name }}</a></li>
+        </ol>
+        <h1>
+            {{ $meeting->id ? __('Edit meeting') : __('Add new meeting') }}
+        </h1>
+    </x-slot>
+
+    <!-- Form Validation Errors -->
+    @include('partials.validation-errors')
+
+    <form class="stack" action="{{ localized_route('meetings.store', $engagement) }}" method="post" novalidate>
+        @csrf
+
+        <x-translatable-input name="title" :label="__('Title of meeting')" :short-label="__('meeting title')" :model="$meeting" />
+
+        <hr class="divider--thick" />
+
+        <h2>{{ __('Time and date') }}</h2>
+
+        <div class="field">
+            <livewire:date-picker name="date" :label="__('Date')" minimumYear="2022" :value="old('date', $meeting->date?->format('Y-m-d'))" />
```

Comment:
Do you want the minimum year to be the current year, or always 2022?

Reply:
You're right, should be current year.